charts/stable/jellyfin/templates/_broadcastProxy.tpl

```diff
@@ -0,0 +1,54 @@
+{{- define "broadcastProxy.container" -}}
+{{- $autodiscoverySvcFqdn := printf "%v-autodiscovery" (include "tc.v1.common.lib.chart.names.fullname" $) -}}
+enabled: true
+primary: true
+imageSelector: broadcastProxyImage
+securityContext:
+  runAsUser: 0
+  runAsGroup: 0
```

Comment:
does this have to run as root and doesn't it need extra caps isntead/aswell?

Reply:
Good catch. It doesn't. Doesn't even need a writable FS. I will ammend